src/mozillavpn.cpp

```diff
@@ -659,6 +660,25 @@ void MozillaVPN::removeDevice(const QString& deviceName) {
   }));
 }
 
+void MozillaVPN::submitFeedback(const QString& feedbackText, const qint8 rating, const QString& category) {
+  logger.log() << "Submit Feedback";
+
+  QString* buffer = new QString();
+  QTextStream* out = new QTextStream(buffer);
+
+  serializeLogs(out, [this, out, buffer, feedbackText, rating, category]{
+    Q_ASSERT(out);
+    Q_ASSERT(buffer);
+
+    delete out;
+    scheduleTask(new TaskSendFeedback(feedbackText, *buffer, rating, category));
+    delete buffer;
```

Comment:
Why a task? You could send it directly, I guess. But, ignoring this part, please write a comment saying that the content of the "buffer" is copied by the TaskSendFeedback CTOR.

Reply:
I saw that a lot of other neworkrequest are using tasks so I wanted to use the same pattern.

Reply:
Sounds good to me. Just write a comment about the lifetime of buffer. Something like:
`TaskSendFeedback CTOR copies the content of buffer. We can delete it immediately  after.` Or something better.